test/e2e/parallel/registration_service_test.go

```diff
@@ -49,6 +52,65 @@ func TestLandingPageReachable(t *testing.T) {
 	assert.Equal(t, http.StatusOK, resp.StatusCode)
 }
 
+func TestRegistrationServiceMetricsEndpoint(t *testing.T) {
+
+	// given
+	await := WaitForDeployments(t)
+	t.Parallel()
+
+	t.Run("not available from default route", func(t *testing.T) { // make sure that the `/metrics`` endpoint is NOT reachable with the default route
+		// given
+		route := await.Host().RegistrationServiceURL
+		req, err := http.NewRequest("GET", route+"/metrics", nil)
+		require.NoError(t, err)
+		// when
+		resp, err := httpClient.Do(req) // nolint:bodyclose // see `defer Close(t, resp)`
+		// then
+		require.NoError(t, err)
+		defer Close(t, resp)
+		assert.Equal(t, http.StatusNotFound, resp.StatusCode)
+	})
+
+	t.Run("available from a custom route", func(t *testing.T) { // create a route for to expose the `registration-service-metrics` svc
+		// given
+		route := &routev1.Route{
```

Comment:
just to make sure I understand this correctly - it's just for verifying the endpoind ? there will not be a route for this in prod right ?

Reply:
got it. thanks for the confirmation!